BIPs/2024-W23/BIP-613/inputs.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "chain_map": {
+      "mode": 34443,
+      "fraxtal": 252
+    },
+    "deployments": [
+      "weighted-pool-v4",
+      "composable-stable-pool-v6",
+      "composable-stable-pool-v5",
+      "20230409-erc4626-linear-pool-v4",
+      "20230409-gearbox-linear-pool-v2",
+      "yearn-linear-pool-v2",
+      "20230410-aave-linear-pool-v5",
+      "20230411-managed-pool-v2"
+    ],
+    "function_caller_map": {
+      "setSwapFeePercentage(uint256)": ["multisigs/lm"],
+      "startAmplificationParameterUpdate(uint256,uint256)": ["multisigs/lm"],
+      "stopAmplificationParameterUpdate()": ["multisigs/lm"],
+      "/pause()": ["multisigs/emergency"],
```

Comment:
is this slash here correct? it seems to interfere with at least the authorizer lookup

Reply:
yeah this uses search_unique() in bal_addresses.  There's also unpause which makes pause not unique but /pause unique.